lib/lynx/d_s_l.rb

```diff
@@ -18,14 +18,14 @@ def create_database
       Command::Basic.new(@config)
         .mysql
         .authorize
-        .sql("create database #{@config.database}")
+        .sql("create database if not exists #{@config.database}")
     end
 
     def drop_database
       Command::Basic.new(@config)
         .mysql
         .authorize
-        .sql("drop database #{@config.database}")
+        .sql("drop database if exists #{@config.database}")
```

Comment:
I think we can make this backwards compatible with new `drop_database_if_exists` and `create_database_if_not_exists` methods.